### ISSUE TITLE
[ffmpeg] build failure

### ISSUE THREAD

**Issue by rkitover** [microsoft/vcpkg#9427]
**Host Environment**
- OS: Windows 10
- Compiler: Visual Studio 2019 x64

**To Reproduce**
`./vcpkg install ffmpeg:x64-windows-static`

**Failure logs** 

Console:

```
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:72 (message):
    Command failed: TASKKILL /F /IM gpg-agent.exe /fi "memusage gt 2"
    Working Directory: C:/Users/rkitover/src/vcpkg/buildtrees/ffmpeg/src/n4.2-02d8c63f80
    Error code: 1
    See logs for more information:
      C:\Users\rkitover\src\vcpkg\buildtrees\ffmpeg\-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_acquire_msys.cmake:119 (vcpkg_execute_required_process)
  ports/ffmpeg/portfile.cmake:43 (vcpkg_acquire_msys)
  scripts/ports.cmake:94 (include)
```

Error log:

```
ERROR: The process "gpg-agent.exe" with PID 16936 could not be terminated.
Reason: Access is denied.
```

**Comment by JackBoosY** [comment]
Can you please close your antivirus software and try again?

**Comment by rkitover** [comment]
All antivirus software is turned off, including defender.

**Comment by rkitover** [comment]
To get it to build for me, I just ran the `taskkill` command from an administrator powershell.

**Comment by JackBoosY** [comment]
So did it work?

**Comment by rkitover** [comment]
Yes, once I killed `gpg-agent` from an administrator powershell the port installed fine.

The question is why I would get "access denied" from a normal user powershell session for this.

**Comment by JackBoosY** [comment]
I don't know, vcpkg should work with powershell with normal permissions.

**Comment by cenit** [comment]
Is it possible that you have a gpg-agent.exe running in background due to whatever service you need (not related to vcpkg), the vcpkg script is too much generic and tries to kill any service matching with that name (in this case an executable that is running also with higher privileges)? @JackBoosY

**Comment by JackBoosY** [comment]
@cenit Good point! If true, this script should be fixed.

**Comment by rkitover** [comment]
I am just using gpg4win from chocolatey for git and such.

**Comment by rkitover** [comment]
@cenit @JackBoosY I'm sorry, I realized this is my fault.

What happened is, I have my msys2 shortcut set to run as administrator, I was using this for symlink support, but this is no longer necessary, I just tried and it works just fine with normal permissions.

So one time I was using git in msys2 before I was using it in a normal powershell session and so it started `gpg-agent.exe` from gpg4win in elevated mode, and later the port script from a normal powershell session did not have the rights to kill it.

This can be closed, but I'd suggest that if killing this process is not integral to the success of the build, the error be changed to a warning, I can do a PR for this if you like.
